docs/CODEOWNERS

```diff
@@ -21,8 +21,8 @@
 # ArchipIDLE
 /worlds/archipidle/ @LegendaryLinux
 
-# Sudoku (BK Sudoku)
-/worlds/bk_sudoku/ @Jarno458
+# Sudoku (APSudoku)
+/worlds/apsudoku/ @EmilyV99
```

Comment:
Very minor nitpick, but would it be possible to move this up so it maintains alphabetical order by path?